chromium/chromium.dockerfile

```diff
@@ -31,15 +37,12 @@ RUN cd /tmp \
 RUN sudo sed -i "s/-w \/home\/user/-w \/home\/user\/chromium\/src/" /etc/supervisord.conf
 
 # Update Chromium third_party repos and run pre-compile hooks.
-WORKDIR src
+WORKDIR /home/user/chromium/src
 RUN gclient runhooks --jobs=18
 
 # Configure Chromium build.
 RUN gn gen out/Default --args="enable_nacl=false is_component_build=true"
 
-# Build Chromium. (This takes hours!)
-#RUN ninja -C out/Default chrome -j18
```

Comment:
Removing this makes me sad, because I'm still hoping to somehow include a pre-compiled Chromium in the Docker image.

But CircleCI times out way before Chromium build even reaches halfway, so fair enough, let's remove this line. (Chromium is still being built on-premise in the `chromium-update.dockerfile`).